resources/ajax_htmldata/getIssuesList.php

```diff
@@ -80,7 +80,7 @@ function generateIssueHTML($issue,$associatedEntities=null) {
 
 			if(count($orgIssues) > 0) {
 				foreach ($orgIssues as $issue) {
-					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->organizationID,"entityType"=>1)));
+					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->primaryKey,"entityType"=>1)));
```

Comment:
After more research, it's what I originally thought.

When the resources module is using the organizations module, the only safe way to access organization data is through the Resource class's getOrganization****() methods because the resource module's Organization class doesn't know about the other database or the additional table fields. 

This doesn't affect the getIssues method on the resources' Organization class, because all Issues live in the resource module's database.

Ideally, the resource module's Organization class would be able to model both the resource and organization module's Organization data, depending on Coral configuration.

I implemented a similar solution for the Issue class in the organization module:
https://github.com/Coral-erm/Coral/commit/a6312f18810efe2cb249aceba84b570eddf1ac0e

But that's beyond the scope of this PR.